ui/app/styles/components/replication-dashboard.scss

```diff
@@ -70,9 +79,9 @@
         grid-row: 3/3;
       }
     }
-  }
 
-  .has-text-highlight {
-    color: $yellow-500;
+    .has-border-danger {
```

Comment:
a lot of the css classes you've set up that only have 1 property (like this one and `has-margin-bottom`) could (and should!) be reused in various places throughout the app, much like the rest of the bulma classes. what do you think about moving these classes into something like `styles/utils/_bulma-modifiers.scss` or something similar? 

an alternative idea if you feel like delving more into sass ~ the `has-margin-[top or bottom]-[size]` could be even more reusable as a [sass mixin](https://sass-lang.com/documentation/at-rules/mixin#arguments). this way in the html markup, you could keep your class names tied to the element (i.e. `<div class="replication-table">`) and extend the mixin from the sass file like so:

```
.replication-table {
  @include has-margin(bottom, $spacing-s)
}

.some-other-element {
  @include has-margin(top, $spacing-xs)
}
```
this nice thing about this approach is that it follows the pattern of keeping elements in html & describing their presentation in sass files. hopefully that makes sense! [here's a lil tutorial](https://scotch.io/tutorials/how-to-use-sass-mixins) on mixins in case you're curious! now that i think about it, we should probably make a css grid mixin since we're starting to use it through the app... 🤔 

part of the reason i bring this idea up is that it seems like a pattern we could reuse going forward. additionally, i recall hearing that the general frontend team is trying to move away from using bulma, though full disclosure, i don't remember why (we could ask the other frontend folks). in fact, i'm just going to summon @DingoEatingFuzz to see if he has any advice on whether to add more custom bulma classes, mixins, or to keep these classes you've defined here as they are. what say ye, michael? 😄

Reply:
For this PR (after discussing with Noelle), we are moving these classes to the `helper.scss` file.  However, we have slotted in some discussions about this topic with design and other frontend teams.

Reply:
I have been summoned!

I'm sure we'll have a great discussion at our guild meeting, but some context and recommendations:

1. The long-term plan is to have a `structure-css` library (as part of `structure-core`) that will make Bulma redundant.
2. IMO, the best way to work towards this is to limit coupling to Bulma where possible. This means making your own net-new CSS with its own modifier classes rather than depending on Bulma's modifiers or extending Bulma's collection of modifiers. This is inline with @noelledaley's suggestion of using mixins. 
3. Generally speaking, I think global modifier classes are an anti-pattern. It makes it really hard to constrain every visual state a component can be in if you have to think about how the component's styles interact with all the global modifiers. Even common modifiers such as danger/warning/success should be mixed into a components scoped styles. (This is my opinion, not necessarily saying this is how `structure-css` will end up).

I look forward to discussing with the group!